treeproxyfilter.cpp

```diff
@@ -41,7 +41,7 @@ bool TreeProxyFilter::filterAcceptsRow(int sourceRow, const QModelIndex &sourceP
 				}
 			}
 			QString key = sourceModel()->data(index, filterRole()).toString();
-			return key.contains(filterRegExp());
+            return key.contains(filterRegularExpression());

```

Comment:
The tabs->spaces change _here_, OTOH, causes this statement to be severely misaligned.

Reply:
Yeah, thanks. I changed my editor for moving forward.